models/DataObject/Service.php

```diff
@@ -61,6 +61,15 @@ class Service extends Model\Element\Service
      */
     protected static $systemFields = ['o_path', 'o_key', 'o_id', 'o_published', 'o_creationDate', 'o_modificationDate', 'o_fullpath'];
 
+    /**
+     * TODO Bc layer for bundles to support both Pimcore 10 & 11, remove with Pimcore 12
+     * @var string[]
+     */
+    protected static array $versionDependentSystemFields = ['id', 'parentId', 'type', 'key', 'path', 'index', 'published',
```

Comment:
```suggestion
    private const array BC_VERSION_DEPENDENT_DATABASE_COLUMNS = ['id', 'parentId', 'type', 'key', 'path', 'index', 'published',
```
WDYT about that? 
And also in general the naming schema.

Reply:
Fine with me, changed name and type (const) of the property and the method name